packages/fxa-payments-server/src/routes/Subscriptions/ActionButton.tsx

```diff
@@ -0,0 +1,110 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useContext } from 'react';
+import { Localized } from '@fluent/react';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { Customer } from '../../store/types';
+import AppContext from '../../lib/AppContext';
+
+import * as PaymentProvider from '../../lib/PaymentProvider';
+
+export type ActionButtonProps = {
+  customer: Customer;
+  onRevealUpdateClick: () => void;
+  revealFixPaymentModal: () => void;
+};
+
+export const ActionButton = ({
+  customer,
+  onRevealUpdateClick,
+  revealFixPaymentModal,
+}: ActionButtonProps) => {
+  const {
+    billing_agreement_id,
+    payment_provider,
+    paypal_payment_error,
+  } = customer;
+
+  const { config } = useContext(AppContext);
+
+  const { apiUrl } = config.paypal;
+
+  const stripeActionButton = () => (
+    <button
+      data-testid="reveal-payment-update-button"
+      className="settings-button"
+      onClick={onRevealUpdateClick}
+    >
+      <Localized id="pay-update-change-btn">
+        <span className="change-button" data-testid="change-button">
+          Change
+        </span>
```

Comment:
Can you adjust the font style/size/hover effect of these Change/Modify buttons? E.g. I just noticed the font color turns blue on hover instead of the background turning a darker grey (and the font looks a little smaller) compared to the "Cancel" button below.

Reply:
Updated font colors, the size and other things are correct to the design, the other elements of the page will be updated in paypal phase 2